engine/classes/Elgg/Email.php

```diff
@@ -247,38 +247,4 @@ protected static function prepareTo($to) {
 
 		return $to;
 	}
-
-	/**
-	 * Parses strings like "Evan <evan@elgg.org>" into name/email objects.
-	 *
-	 * This is not very sophisticated and only used to provide a light BC effort.
-	 *
-	 * @param string $contact e.g. "Evan <evan@elgg.org>"
-	 *
-	 * @return Address
-	 */
-	public static function fromString($contact) {
-		$containsName = preg_match('/<(.*)>/', $contact, $matches) == 1;
-		if ($containsName) {
-			$name = trim(substr($contact, 0, strpos($contact, '<')));
-			return new Address($matches[1], $name);
-		} else {
-			return new Address(trim($contact));
-		}
-	}
-
-	/**
-	 * Format an email address and name into a formatted email address
-	 *
-	 * eg "Some name <someone@example.com>"
-	 *
-	 * @param string $email the email address
-	 * @param string $name  the name
-	 *
-	 * @return string
-	 */
-	public static function getFormattedEmailAddress($email, $name = null) {
```

Comment:
same

Reply:
is documented as it came from a different class, and that change is listed